firewallrule.go

```diff
@@ -64,7 +64,8 @@ func (c *Client) CreateFirewallRule(dcID string, serverID string, nicID string,
 	return ret, err
 }
 
-//UpdateFirewallRule updates a firewall rule
+// UpdateFirewallRule updates a firewall rule.
+// You need to pass all wanted properties, not just those you want to change.
```

Comment:
Why is this even a PATCH then?

Reply:
It does not matter if PATCH or PUT. As consumer there won't be a difference and the API cannot be used otherwise.